File: c++/Mercantil/src/Person.cpp
#include "../src/include/Person.h"

// Construtor da classe person
Person::Person(std::string name) : name(name) {}

// Getters e setters da classe Person
std::string Person::getName() const { return name; }
void Person::setName(std::string name) { this->name = name; }

File: java/Motoca/Person.java
package Motoca;

public class Person {
    private int age;
    private String name;
    
    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    public int getAge() { return age; }
    public String getName() { return name; }

    public String toString() {
        return "(" + this.name + ":" + this.age + ")";
    }
}